### DISCUSSION TITLE
如何修改 openapi 中的标识符号

### DISCUSSION THREAD

**Discussion by tpc-ht** [salvo-rs/salvo#602] [Q&A]
目前想根据 openapi.json 生成前端接口函数与类型。但是项目生成的 json 并没有满足需求
[最小实现](https://github.com/tpc-ht/salvo-swagger-test)
类型名如何自定义生成
![image](https://github.com/salvo-rs/salvo/assets/59469126/519fdc3c-2334-451a-83f9-2955eabc0913)

**Comment by tpc-ht** [comment]
该如何修改了？

**Reply by chrislearn** [reply]
不是让你用 symbol 试试看

**Reply by tpc-ht** [reply]
哈哈哈，是我当时没理解意思。弄好了，谢谢大佬

**Comment by tpc-ht** [comment]
发现普通的 struct 没问题，但是基于泛型的通用 struct，有问题。
![image](https://github.com/salvo-rs/salvo/assets/59469126/bb25ba77-7e26-4db6-96e5-693c7bf2bc44)

这个是否可以在控制层动态设置 标识
![image](https://github.com/salvo-rs/salvo/assets/59469126/d5348948-d8ed-4cb7-b3e0-73b62ef9e04a)

**Reply by chrislearn** [reply]
已经修复 #642
